ElfUtils/ElfFileTest.cpp

```diff
@@ -162,7 +165,19 @@ TEST(ElfFile, CreateFromBuffer) {
 
   auto elf_file =
       ElfFile::CreateFromBuffer(test_elf_file, buffer.data(), buffer.size());
-  ASSERT_NE(elf_file, nullptr);
-  EXPECT_EQ(elf_file->GetBuildId(),
+  ASSERT_TRUE(elf_file) << elf_file.error().message();
+  EXPECT_EQ(elf_file.value()->GetBuildId(),
             "d12d54bc5b72ccce54a408bdeda65e2530740ac8");
 }
+
+TEST(ElfFile, FileDoesNotExist) {
+  std::string executable_path = Path::GetExecutablePath();
+  std::string file_path = executable_path + "/testdata/does_not_exist";
+
+  auto elf_file = ElfFile::Create(file_path);
+  ASSERT_FALSE(elf_file);
+  EXPECT_EQ(elf_file.error().message(),
+            absl::StrFormat(
+                "Unable to create elf-file \"%s\": No such file or directory",
```

Comment:
Do we want a test that precisely compares the error message? It would fail as soon as one character is changed in the error and requires the two strings to be kept in sync.

Reply:
I kind of agree. If you want to ensure that certain elements are correct, maybe we should just "grep" for them?

Reply:
done.